spacy/training/corpus.py

```diff
@@ -19,14 +20,19 @@
 FILE_TYPE = ".spacy"
 
 
+class ReaderProtocol(Protocol):
+    def __call__(self, nlp: "Language") -> Iterable[Example]:
```

Comment:
Sorry, commented on the wrong PR the first time.

I thought `Iterator[Example]` was correct throughout here?

Reply:
And now this comment is also buried in a closed PR, hmm...